### PR TITLE
create_release.py: Push resulting branch to origin

### PR DESCRIPTION
**Pull request by sbc100**
Also, set the upstream when creating the new branch.

See https://github.com/emscripten-core/emsdk/pull/1255

### CODE REVIEW

**Review by dschuff — COMMENTED** [review]
ah, so I just realized that I use the emscripten repo inside an emscripten-releases checkout. this means its 'origin' is actually chromium.googlesource.com/emscripten rather than github (I just have another remote to push to github). So this won't work for me.
One option would be to make this overridable via an env var or something. but let me also check whether gclient still works if I rename my origin remote.

**Comment by dschuff** [comment]
OK, yeah gclient sync seems to work fine with the github remote as 'origin'.

File: tools/maint/create_release.py [hunk 1]
```diff
@@ -59,15 +59,15 @@ def main():
...
+  subprocess.check_call(['git', 'checkout', '-b', branch_name, 'origin/main'], cwd=root_dir)
 
   # Create auto-generated changes to the new git branch
   subprocess.check_call(['git', 'add', '-u', '.'], cwd=root_dir)
   subprocess.check_call(['git', 'commit', '-m', f'Mark {release_version} as released'], cwd=root_dir)
+  print('New release created in branch: `%s`' % branch_name)
 
-  print('New relase created in branch: `%s`' % branch_name)
-
-  # TODO(sbc): Maybe create the tag too, and even push both to `origin`?
```
Comment (dschuff) [inline tools/maint/create_release.py:70]: is creating the tag something we want to do in this script, either in this PR or leave the TODO for that?

**Review by dschuff — APPROVED** [review]

File: tools/maint/create_release.py (hunk 1 above)
Comment (sbc100) [reply tools/maint/create_release.py:70]: Oh yeah, I was wasn't sure about that.  I leave the TODO for now.

**Pull request merged**